app/resources/views/components/gameboard/investitionen/investitionen-immobilien-modal.blade.php

```diff
@@ -0,0 +1,158 @@
+@extends ('components.modal.modal', ['closeModal' => "toggleImmobilienModal()"])
+
+@use('Domain\CoreGameLogic\Feature\Konjunkturphase\State\ImmobilienPriceState')
+
+@props([
+    'immobilien' => [],
+    'immobilienOwned' => [],
+])
+
+@section('icon')
+    <x-gameboard.phase-icon />
+@endsection
+
+@section('title')
+    @if ($this->isBuyImmobilieVisible)
+        <span>
+            Kauf - Immobilie <i class="icon-immobilien" aria-hidden="true"></i>
+        </span>
+    @elseif ($this->isSellImmobilieVisible)
+        <span>
+            Verkauf - Immobilie <i class="icon-immobilien" aria-hidden="true"></i>
+        </span>
+    @else
+        <span>
+            Immobilien <i class="icon-immobilien" aria-hidden="true"></i>
+        </span>
+        <span class="font-size--base">
+            Investitionen
+        </span>
+    @endif
+@endsection
+
+@section('content')
+    @if ($this->isBuyImmobilieVisible)
+        <div class="immoblien">
+            @foreach($immobilien as $immobilie)
+                <div @class(["card", "card--disabled" => !$this->canBuyImmobilie($immobilie->getId())->canExecute])>
+                    <h4 class="card__title">{{ $immobilie->getTitle() }}</h4>
+                    <div class="card__content card__content--center">
+                        <div class="resource-changes">
+                            <div class="resource-change">
+                                {!! $immobilie->getAnnualRent()->formatWithIcon() !!}
+                            </div>
+                        </div>
+                        <span class="font-size--sm">Jährliche Miete</span>
+                        <x-gameboard.resourceChanges.resource-changes :resource-changes="$immobilie->getResourceChanges()" />
+                        <span class="font-size--sm">Kaufpreis</span>
+
+                        <button type="button"
+                            @class([
+                                "button",
+                                "button--type-primary",
+                                "button--disabled" => !$this->canBuyImmobilie($immobilie->getId())->canExecute,
+                                $this->getPlayerColorClass()
+                            ])
+                            wire:click="buyImmobilie('{{ $immobilie->getId()->value }}')"
+                        >
+                            Kaufen
+                            <div class="button__suffix">
+                                <div>
+                                    <i class="icon-minus text--danger" aria-hidden="true"></i><i class="icon-zeitstein" aria-hidden="true"></i>
+                                    <span class="sr-only">, kostet einen Zeitstein</span>
+                                </div>
+                            </div>
+                        </button>
+                    </div>
+                </div>
+            @endforeach
+        </div>
+    @elseif ($this->isSellImmobilieVisible)
+        @if ($immobilienOwned)
```

Comment:
Hm. Lieber was expliziteres:
```suggestion
        @if (count($immobilienOwned) > 0)
```